roxie/ccd/ccdserver.cpp

```diff
@@ -1203,11 +1206,18 @@ class CRoxieServerActivity : public CInterface, implements IRoxieServerActivity,
                         dependencies.item(idx).stopSink(dependencyIndexes.item(idx));
                 }
                 if (input)
-                    input->stop(aborting);
+                    input->stop();
             }
         }
     }
 
+    virtual void abort()
+    {
+        aborted = true;
+        // MORE - should abort dependencies here?
+        stop();
```

Comment:
I am not sure if abort() should call stop(), or whether they should be separate.

E.g., you may want to call abort() on all activities when a process is guillotined.